packages/css/src/page-menu/page-menu.scss

```diff
@@ -69,6 +69,7 @@
 .amsterdam-page-menu__link:hover,
```

Comment:
I realise after reading the interactive elements that I did not include a focus-visible state with the token --amsterdam-focus-outline-offset.